utils/make-pkgconfig.swift

```diff
@@ -56,8 +56,14 @@ func makeFile() throws {
   }
   let cllvmPath = pkgConfigDir.appendingPathComponent("cllvm.pc")
 
+  let brewLLVMConfig: () -> String? = {
+    guard let brew = which("brew") else { return nil }
+    guard let brewPrefix = run(brew, args: ["--prefix"]) else { return nil }
+    return which(brewPrefix + "/opt/llvm/bin/llvm-config")
```

Comment:
Does `brew —prefix` let you get down to package-level locations so you can strip even more of the constant string?